crowbar_framework/app/models/service_object.rb

```diff
@@ -1642,9 +1662,9 @@ def equal_or_match_version(value, maybe_regexp)
     if maybe_regexp.start_with?('/') && maybe_regexp.end_with?('/')
       Regexp.new(maybe_regexp[1..-2]).match(value)
     elsif op_value.length > 0
-      cmp.call(op_value.first.first, op_value.first.last)
+      oper.call(op_value.first.first, op_value.first.last)
```

Comment:
For the record, reading this line is actually hard without reparsing the previous lines. Might be good to use some (arguably useless) variables just to get some good variable names for `op_value.first`, or `op_value.first.first`, etc.